modules/db/main.tf

```diff
@@ -0,0 +1,43 @@
+
+resource "azurerm_postgresql_flexible_server" "aap" {
+  name                   = "${var.deployment_id}-psqlflexibleserver"
+  resource_group_name    = "${var.resource_group}"
+  location               = "${var.location}"
```

Comment:
```suggestion
  location               = var.location
```

Reply:
Why do you suggest this change?  Thought we would need to quote strings

Reply:
tflint is suggesting that its going to be deprecated 
```
Warning: [Fixable] Interpolation-only expressions are deprecated in Terraform v0.12.14 (terraform_deprecated_interpolation)

  on main.tf line 55:
  55:   address_prefixes     = ["${var.infrastructure_vpc_subnet_cidr_postgres}"]
 ```

Reply:
changed